com.microsoft.mrtk.uxcore/Keyboard/NonNative/NonNativeKeyboard.cs

```diff
@@ -246,11 +246,6 @@ private set
         /// </summary>        
         private bool isRecording = false;
 
-        /// <summary>
-        /// On the first recording
-        /// </summary>        
-        private bool firstRecording = true;
```

Comment:
yay!

Reply:
haha it was annoying me so I slipped this in